### PR TITLE
Update resources-data.ts

### PR DESCRIPTION
**Pull request by randy-r**
Hello Stencil Team,

Please let me know if I can add those resources to the site.
Awesome work by the way!

Thanks,
Randy.

### CODE REVIEW

File: src/components/resources-page/resources-data.ts [hunk 1]
```diff
@@ -160,6 +162,20 @@ export const RESOURCES = {
       imgPath: '/assets/img/demos/demo-revogrid',
       demoUrl: 'https://revolist.github.io/revogrid',
       sourceUrl: 'https://github.com/revolist/revogrid',
+    },
+    {
+      title: 'Stedux',
+      description: 'Example website for the using the Stedux library',
```
Comment (tanner-reits) [inline src/components/resources-page/resources-data.ts:168]:
```suggestion
      description: 'Example website for using the Stedux library',
```

File: src/components/resources-page/resources-data.ts [hunk 2]
```diff
@@ -160,6 +162,20 @@ export const RESOURCES = {
...
+      imgPath: '/assets/img/demos/demo-stedux',
+      demoUrl: 'https://randy-r.github.io/stedux',
+      sourceUrl: 'https://github.com/randy-r/stedux',
+    },
+    {
+      title: 'Stenimator',
+      description: 'Example website for the using the Stenimator library',
```
Comment (tanner-reits) [inline src/components/resources-page/resources-data.ts:175]:
```suggestion
      description: 'Example website for using the Stenimator library',
```

File: src/components/resources-page/resources-data.ts [hunk 3]
```diff
@@ -29,7 +29,9 @@ export const RESOURCES = {
     { title: 'Badge Component', url: 'https://github.com/ftonato/stenciljs-components/tree/master/badge-component' },
     { title: 'Progress Component', url: 'https://github.com/ftonato/stenciljs-components/tree/master/progress-component' },
     { title: 'A small collection of Awesome Stencil UI components', url: 'https://github.com/dlucidone/awesome-ui' },
-    { title: 'Soft Components (Neumorphism design inspired web components)', url: 'https://soft.components.rocks' }
+    { title: 'Soft Components (Neumorphism design inspired web components)', url: 'https://soft.components.rocks' },
+    { title: 'Stedux: Redux for Stencil', url: 'https://github.com/randy-r/stedux' },
+    { title: 'Stenimator: Animation coordintor for elements leaving the DOM', url: 'https://github.com/randy-r/stenimator' }
```
Comment (rwaskiewicz) [inline src/components/resources-page/resources-data.ts:34]:
```suggestion
    { title: 'Stenimator: Animation coordinator for elements leaving the DOM', url: 'https://github.com/randy-r/stenimator' }
```

**Review by rwaskiewicz — CHANGES_REQUESTED** [review]
Thanks @randy-r! 

The team and I had a few small nitpicks we found while reviewing the PR. Could you please take a look when you get a chance? Thanks again!

**Comment by randy-r** [comment]
Hi @rwaskiewicz !

I should thank you and the team!
I have addressed your suggestions and adjusted the image size as they were a bit off.

Cheers!

**Review by rwaskiewicz — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**